test/e2e/app-dir/ppr-navigations/simple/per-segment-prefetching.test.ts

```diff
@@ -0,0 +1,46 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('per segment prefetching', () => {
+  if ((global as any).isNextDev) {
+    test('ppr is disabled in dev', () => {})
+    return
+  }
+
+  const { next } = nextTestSetup({
```

Comment:
```suggestion
  const { next, isNextDev, isNextDeploy } = nextTestSetup({
```
Can grab these from here instead of global now